build.gradle

```diff
@@ -1,3 +1,14 @@
+buildscript {
+  repositories {
+    maven {
+      url "https://maven.global.square/artifactory/square-public"
```

Comment:
this shouldn't be here

Reply:
correct, either swap in `mavenCentral()` here:
```
repositories {
  mavenCentral()
}
```

or add to the `plugins` block below:

```
plugins {
  ...
  id 'com.squareup.wire' version "3.0.0-alpha01"
}
```